docs/sts/ldap.md

```diff
@@ -22,6 +22,24 @@ applicable policies on a user (these are the policies associated with
 the groups along with the policy on the user if any) to check if the
 request should be allowed or denied.
 
+**Table of Contents**
+
+- [Configuring AD/LDAP on MinIO](#configuring-adldap-on-minio)
+    - [Variable substitution in AD/LDAP configuration strings](#variable-substitution-in-adldap-configuration-strings)
+- [Notes on configuring with Microsoft Active Directory (AD)](#notes-on-configuring-with-microsoft-active-directory-ad)
+- [STS API Parameters](#sts-api-parameters)
+    - [Request Parameters](#request-parameters)
+        - [LDAPUsername](#ldapusername)
+        - [LDAPPassword](#ldappassword)
+        - [Version](#version)
+        - [Policy](#policy)
+    - [Response Elements](#response-elements)
+    - [Errors](#errors)
+    - [Sample Request](#sample-request)
+    - [Sample Response](#sample-response)
+    - [Testing](#testing)
+- [Managing User/Group Access Policy](#managing-usergroup-access-policy)
+
 ## Configuring AD/LDAP on MinIO
```

Comment:
Can you do the TOC for other STS implementations as well? @donatello as an umbrella doc change?

Reply:
Sure, I'll remove it from this PR and send a new one with only TOCs.